app/controllers/events_controller.rb

```diff
@@ -102,6 +103,13 @@ def set_event
       @event = Event.find(params[:id])
     end
 
+    def set_playercount_per_team
+      if event_params[:player_type] == "single" || @event.type == 'Rankinglist'
+        @event.min_players_per_team = 1
+        @event.max_players_per_team = 1
```

Comment:
is this reasonable? Should the feature be disabled or is it not possible that there will be ranking lists for teams as well? If, at some point (or even already now), it should be possible that teams participate in a ranking list, this needs to be changed.

Reply:
As far as I know, ranking lists are single-player-only.